### PR TITLE
feat: set 'disableAnalytics true'

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium/issues/15274
It is reasonable to disable this to prevent unexpected fatal error.

### CODE REVIEW

File: lib/uiautomator2.js [hunk 1]
```diff
@@ -266,6 +266,8 @@ class UiAutomator2Server {
       cmd.push('-e', 'DISABLE_SUPPRESS_ACCESSIBILITY_SERVICES', this.disableSuppressAccessibilityService);
     }
     cmd.push(INSTRUMENTATION_TARGET);
+    // Disable Google analytics to prevent possible fatal exception
+    cmd.push('-e', 'disableAnalytics', true);
```
Comment (mykola-mokhnach) [inline lib/uiautomator2.js:270]: shouldn't we push it before setting the target?

Comment (KazuCocoa) [reply lib/uiautomator2.js:270]: oh, that's right

**Review by mykola-mokhnach — APPROVED** [review]

File: lib/uiautomator2.js [hunk 2]
```diff
@@ -265,6 +265,8 @@ class UiAutomator2Server {
     if (_.isBoolean(this.disableSuppressAccessibilityService)) {
       cmd.push('-e', 'DISABLE_SUPPRESS_ACCESSIBILITY_SERVICES', this.disableSuppressAccessibilityService);
     }
+    // Disable Google analytics to prevent possible fatal exception
+    cmd.push('-e', 'disableAnalytics', true);
```
Comment (mykola-mokhnach) [inline lib/uiautomator2.js:269]: I assume `true` should be a string

Comment (KazuCocoa) [reply lib/uiautomator2.js:269]: link: https://github.com/appium/appium-espresso-driver/pull/666#discussion_r614686105

**Pull request merged**